core/src/main/java/org/sapia/ubik/mcast/EventConsumer.java

```diff
@@ -305,12 +316,22 @@ private void notifyAsyncListeners(RemoteEvent evt) {
     if (lst.getApproximateSize() == 0) {
       log.debug("No listener for event: %s", evt.getType());
     }
```

Comment:
I had removed this in my initial fix. I adopted a dynamic approach with a counter rather than explicitly counting the number of handlers before hand. Reinserting this piece of code makes now the check for zero listener twice. Just pick one way and remove the other.